portal-docs/observability.md

```diff
@@ -8,6 +8,8 @@
 * [Observe Application Metrics](#Observe-Application-Metrics)
 * [Tracing Requests](#Tracing-Requests)
 * [View Logs](#View-Logs)
+   * [Correlate logs with throughput and latency graphs](#correlate-logs-with-throughput-and-latency-graphs)
```

Comment:
We seem to be using title-style capitalization for some of the toc items. Better to follow the same for all toc items.